src/components/dropdown-menu/DropdownMenuItemLink.tsx

```diff
@@ -38,18 +39,27 @@ export function DropdownMenuItemLink({
 
   const isExternal = isExternalProp || toAsString.startsWith('http');
 
+  const externalSuffix = isExternal ? <StyledIconLinkExternal /> : undefined;
+
+  let composedSuffix = suffix ?? externalSuffix;
+
+  if (suffix && externalSuffix) {
+    composedSuffix = (
+      <span className="sw-flex sw-gap-2 sw-items-center">
+        {suffix}
+        {externalSuffix}
+      </span>
+    );
+  }
```

Comment:
NP: I'd move this to a component or helper function

Reply:
done